types/three/three-mapcontrols.d.ts

```diff
@@ -0,0 +1,70 @@
+import {Camera, MOUSE, Object3D, Vector3} from 'three'
+
+export class MapControls {
+    constructor(object: Camera, domElement?: HTMLElement)
+
+    object: Camera
+    domElement: HTMLElement | HTMLDocument
+
+    // API
+    enabled: boolean
+    target: Vector3
+
+    enableZoom: boolean
+    zoomSpeed: number
+    minDistance: number
+    maxDistance: number
+    enableRotate: boolean
+    rotateSpeed: number
+    enablePan: boolean
+    maxZoom: number
+    minZoom: number
+    panSpeed: number
+    autoRotate: boolean
+    autoRotateSpeed: number
+    minPolarAngle: number
+    maxPolarAngle: number
+    minAzimuthAngle: number
+    maxAzimuthAngle: number
+    enableKeys: boolean
+    screenSpacePanning: boolean
+    keys: {LEFT: number; UP: number; RIGHT: number; BOTTOM: number}
+    mouseButtons: {ORBIT: MOUSE; ZOOM: MOUSE; PAN: MOUSE}
```

Comment:
These dont seem to be the right object keys for `mouseButtons` https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/MapControls.js#L80